src/storage/admin/RebuildEdgeIndexTask.cpp

```diff
@@ -116,6 +116,7 @@ kvstore::ResultCode RebuildEdgeIndexTask::buildIndexGlobal(GraphSpaceID space,
                     IndexKeyUtils::collectIndexValues(reader.get(), item->get_fields());
                 if (!valuesRet.ok()) {
                     LOG(WARNING) << "Collect index value failed";
+                    iter->next();
```

Comment:
Isn't it necessary here? 
Because the current index is wrong, other indexes are not necessarily wrong

Reply:
It will infinite loop if we go to here.

Reply:
Firstly, there may be multiple indexes, the current index is wrong, `next(),` the following index is not processed.
Secondly, the index is processed on line 132

Reply:
Ah, sorry, I pushed the wrong commit, this has been fixed before. Thx.